ql/src/java/org/apache/hadoop/hive/ql/parse/BaseSemanticAnalyzer.java

```diff
@@ -493,6 +495,10 @@ public static String getUnescapedUnqualifiedTableName(ASTNode node) throws Seman
     return getUnescapedName(node);
   }
 
+  public static boolean getIsRemoteType(){
```

Comment:
a) I think we should call this method isDbRemote() or isDatabaseRemote() instead of getIsRemoteType()
b) this should be either a protected method so its only used by its sub-classes? being doesnt make much sense given the return value is specific to a database but it doesnt take any arguments. Or if we want to make it static, we can add a isDatabaseRemote(Database db) method to a utils function.